packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -199,38 +614,27 @@ class DatePickerInput extends React.Component {
           name={name}
           value={dateValue}
```

Comment:
Is this correct? Should it be `date` after focus is shifted or -prepended, does this need to change?

Reply:
`dateValue` is always the updated value, which includes the updated value after auto prepend as well.
If the date is valid then it will be in ISO format else it will be the raw input.